engines/restic.go

```diff
@@ -364,5 +310,5 @@ func (r *ResticEngine) launchRestic(cmd []string, volumes []*volume.Volume) (sta
 		env[k] = v
 	}
 
-	return r.Orchestrator.LaunchContainer(image, env, r.replaceArgs(cmd), volumes)
+	return r.Orchestrator.LaunchContainer(image, env, r.replaceArgs(append(cmd, config.Restic.CommonArgs...)), volumes)
```

Comment:
Actually, the variable's value of `config.Restic.CommonArgs` is `[]string{"-r s3:http://my-repo --argument1 --argument2"}`. Therefore, you pass this kind of string to Restic: `init --no-cache "-r s3:http://my-repo --argument1 --argument2"`.
That's the reason why Restic says that the backend is invalid.